apps/api/src/app/admin/admin.service.ts

```diff
@@ -640,7 +646,13 @@ export class AdminService {
     return { marketData, count: marketData.length };
   }
 
-  private async getUsersWithAnalytics(): Promise<AdminUsers['users']> {
+  private async getUsersWithAnalytics({
+    take,
+    skip
```

Comment:
Can you swap `skip` and `take` to preserve the alphabetical order please?

Reply:
Done.